src/main/flight/pid.c

```diff
@@ -136,10 +136,21 @@ typedef float (*pidControllerFn)(const pidProfile_t *pidProfile, int axis, float
 float featheredPids(const pidProfile_t *pidProfile, int axis, float errorRate, float dynCi, float currentPidSetpoint);
 float classicPids(const pidProfile_t *pidProfile, int axis, float errorRate, float dynCi, float currentPidSetpoint);
 
+float errorBoostLimit = 1.0f;
+float errorMultiplier = 1.0f;
+
+float featheredPIDboost( float errorRate)
+{
+	float boostedErrorFactor = (errorRate * errorRate) * errorMultiplier;
```

Comment:
How does this deal with the error being a negative value? May it is, just verifying.

Reply:
multiplication.
the boostfactor is positive and the error is positive or negative.
the sign is not a problem.

Reply:
aww my bad i didn't notice that at first. I shouldn't look at code this early in the morning.